system/Log/Handlers/FileHandler.php

```diff
@@ -77,7 +77,8 @@ public function __construct(array $config = [])
 	{
 		parent::__construct($config);
 
-		$this->path = $config['path'] ?: WRITEPATH . 'logs/';
+		$config['path'] = $config['path'] ?? WRITEPATH . 'logs/';
+		$this->path     = $config['path'] ?: WRITEPATH . 'logs/';
```

Comment:
I suggest this as it's less code and uses the same pattern as the other property assignments.

```
    $this->path = empty($config['filePath']) ? WRITEPATH . 'logs/' : $config['filePath'];
```

Reply:
@dafriend implemented.

Reply:
@dafriend I rolled back the changes to use the `??` first, and `?:` later as it can make for whatever reason user create folder named "0", it will still working.

Reply:
@dafriend I just realized that the directory path requires `/` suffix, so "0" is not valid, "0/" is valid. I applied your suggestion.